README.md

```diff
@@ -277,62 +287,137 @@ signIn.renderEl(
     // screen letting them know that an email is on the way.
     if (res.status === 'FORGOT_PASSWORD_EMAIL_SENT') {
       // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
       return;
     }
 
     // The user has started the unlock account flow, and is on the confirmation
     // screen letting them know that an email is on the way.
     if (res.status === 'UNLOCK_ACCOUNT_EMAIL_SENT') {
       // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
+      return;
+    }
+
+    // The user has started the activation flow, and is on the confirmation
+    // screen letting them know that an email is on the way.
+    if (res.status === 'ACTIVATION_EMAIL_SENT') {
```

Comment:
We previously did not document `ACTIVATION_EMAIL_SENT ` as a possible status. I could use some assistance on better documenting the scenario that results in this block being called beyond what I've written so far.

Reply:
I think this happens when using self-service registration. The admin can configure either SMS or email activation I think.